src/components/Console.jsx

```diff
@@ -20,8 +21,12 @@ export default function Console({
   const console = (
     <div className="console__repl output__item">
       {history.map((entry, key) => (
-        // eslint-disable-next-line react/no-array-index-key
-        <ConsoleEntry entry={entry} key={key} />
+        <ConsoleEntry
+          currentCompiledProjectKey={currentCompiledProjectKey}
```

Comment:
I don’t feel strongly about it but I think there’s probably an argument for just doing the logic here and passing an `isActive` prop into `<ConsoleEntry>`

Reply:
Yeah I was thinking that. It seems cleaner but wasn't sure if it was important that the logic was done within the element for which the evaluation was being used

Reply:
There’s definitely no objective answer to that question—it’s just a matter of design:

* The way you’ve got it now makes the `<ConsoleEntry>` more self-contained; the calling component does not need to know anything about active vs. inactive entries. Useful if `<ConsoleEntry>` might be used in a bunch of different contexts, but always have the same logic for active vs. inactive
* The change I’m proposing makes the `<ConsoleEntry>` more flexible—it doesn’t need to know _why_ it might be inactive, it just does what it’s told. Useful if `<ConsoleEntry>` might be used in different contexts with _different_ logic for what makes it active vs. inactive

In this case neither of the practical scenarios seems likely, which is why I don’t feel particularly strongly about it : )